qt5ct.tera

```diff
@@ -0,0 +1,18 @@
+---
+whiskers:
+  version: "2.5.0"
+  matrix:
+    - flavor
+    - accent
+  filename: "themes/Catppuccin-{{ flavor.identifier | capitalize }}-{{accent | capitalize }}.conf"
```

Comment:
Do the filenames need to be capitalised? I'd like to standardise on lowercase in the organisation (i.e. `catppuccin-mocha-mauve.conf`)

```suggestion
  filename: "themes/catppuccin-{{ flavor.identifier }}-{{ accent }}.conf"
```